go.mod

```diff
@@ -0,0 +1,39 @@
+module github.com/redhat-cop/ciao
```

Comment:
Is there a reason the module name is `ciao` instead of `cert-utils-operator`? Please check the imported packages as well in the `main.go` file

Reply:
thanks for spotting this. I think this is what was causing the build issues. It is fixed now.